hc/accounts/tests/test_login.py

```diff
@@ -17,20 +17,26 @@ def test_it_sends_link(self):
         session.save()
 
         form = {"email": "alice@example.org"}
+        resp = self.client.post("/accounts/login/", form)
+        # self.assertEqual(resp.status_code, 302)
```

Comment:
Well, if it is not part of your submitted work, just remove it entirely instead of commenting it out.

**FYI**: HTTP status code `302` indicates a redirect. see [here](https://goo.gl/xYa2Nw)